--- Makefile	2013-03-08 12:39:02.000000000 -0600
+++ Makefile	2013-03-11 12:23:53.000000000 -0500
@@ -8,6 +8,7 @@
 PWD:=$(shell pwd)
 
 DAHDI_MODULES_EXTRA:=$(MODULES_EXTRA:%=%.o) $(SUBDIRS_EXTRA:%=%/)
+DAHDI_MOD_DIR:=extra/dahdi-linux
 
 # If you want to build for a kernel other than the current kernel, set KVERS
 ifndef KVERS
@@ -152,7 +153,7 @@
 	fi
 	build_tools/uninstall-modules dahdi $(KVERS)
 endif
-	$(KMAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=dahdi modules_install
+	$(KMAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=$(DAHDI_MOD_DIR) modules_install
 	[ `id -u` = 0 ] && /sbin/depmod -a $(KVERS) || :
 
 uninstall-modules:
